recipes/abinit/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./configure --prefix=${PREFIX}
+make
```

Comment:
You can use multiple cores do compile this project if you want to
```suggestion
make ${CPU_COUNT}
```